src/engine.ts

```diff
@@ -140,6 +141,12 @@ export default class Engine {
     return this.turnOrder.map(i => this.players[i]);
   }
 
+  playersInTableOrderFromCurrent(): Player[] {
+    let originalOrder = this.players.filter(pl => pl.player >= this.currentPlayer).map(pl => pl.player);
+    originalOrder = originalOrder.concat(this.players.filter(pl => pl.player < this.currentPlayer).map(pl => pl.player));
+    return originalOrder.map(i => this.players[i]);
```

Comment:
```js
return this.players.slice(this.currentPlayer+1).concat(this.players.slice(0, this.currentPlayer)
```

Reply:
by the way, taking care of #73 , nice :+1:

Reply:
much nicer!